core/src/main/java/org/elasticsearch/script/ScriptMode.java

```diff
@@ -24,13 +24,11 @@
 
 /**
  * Mode for a specific script, used for script settings.
- * Defines whether a certain script or category of scripts can be executed or not, or whether it can
- * only be executed by a sandboxed scripting language.
+ * Defines whether a certain script or category of scripts can be executed or not.
  */
 enum ScriptMode {
```

Comment:
Definitely another PR, I started doing it on this one but it turned into a mess so it'd be nice to separate.